app/models/ability.rb

```diff
@@ -0,0 +1,13 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    return unless user.present?
+
+    can :manage, Post, author_id: user.id
+    can :manage, Comment, user_id: user.id
```

Comment:
- [ ] The requirement is that only post owners or admins can manage posts or comments. Regular users who are neither should still be able to read posts, even if they can't delete them. Kindly add the following line here.

  ```
  can :read, :all
  ```
  
  ![image](https://github.com/K0ppai/Blog_Web/assets/54848363/6277b6a9-7194-4a88-963a-a015ce0bc5bb)